roles/kubernetes/node/defaults/main.yml

```diff
@@ -106,7 +106,7 @@ kubelet_healthz_port: 10248
 kubelet_healthz_bind_address: 127.0.0.1
 
 # sysctl_file_path to add sysctl conf to
-sysctl_file_path: "/etc/sysctl.d/99-sysctl.conf"
+sysctl_file_path: "{{ sysctl_file_path }}"
```

Comment:
here needs to be written as `sysctl_file_path: /etc/sysctl.d/99-sysctl.conf` for the default value to avoid
```
Error was a <class 'ansible.errors.AnsibleError'>, original message: An unhandled exception occurred while templating '{{ sysctl_file_path }}'.
Error was a <class 'ansible.errors.AnsibleError'>, original message: An unhandled exception occurred while templating '{{ sysctl_file_path }}'.
Error was a <class 'ansible.errors.AnsibleError'>, original message: recursive loop detected in template string: {{ sysctl_file_path }}"}
```
as https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/1992319421

Reply:
Done.